public/documents/fari-games/charge-srd.md

```diff
@@ -178,9 +178,9 @@ When there's a challenge that needs to be overcome, and there's a risk of failur
 
 The number of dice to roll depends on the `action` the character is doing. The list of possible action can be found in the character creation section.
 
-The action roll acts as double duty. Which means that it resolves what the PC is doing, as well as the NPC.
+In this game, NPCs don't roll for their actions. Instead, an action roll does double-duty: **it resolves the action of the PC as well as any NPCs that are involved**. The action roll tells us the results, and the consequences of an action at the same time.
```

Comment:
If taking above suggestion for "NPCs always succeed unless players intervene", might make sense to carry it through to the SRD something along those lines? 

```suggestion
In this game, NPCs don't roll for their actions. NPCs succeed unless the players attempt to stop them, in which case the action roll does double-duty: **it resolves the action of the PC as well as any NPCs that are involved**. The action roll tells us the results, and the consequences of an action at the same time, encouraging players to be active participants in the story.
```